### PR TITLE
Sharing: Update Twitter Article Previews

### PR DESCRIPTION
**Pull request by jhnstn**
Use real data for twitter preview 
screen shot 2017-03-27 at 5 34 14 pm

**Testing**
- visit http://calypso.localhost:3000/devdocs/blocks/sharing-preview-pane
- click twitter tab

related PR #12490

### CODE REVIEW

File: client/components/share/twitter-share-preview/index.jsx [hunk 1]
```diff
@@ -5,14 +5,15 @@ import React, { PureComponent } from 'react';
...
-		const articleUrl = 'https://styleandgear.com/2016/01/03/how-to-dress-like-a-proper-we-need-some-long-url';
-		const imageUrl = 'https://www.w3schools.com/css/trolltunga.jpg';
+		const {
+			externalProfilePicture,
+			externalProfileUrl,
+			externalDisplay,
+			externalName,
+			message,
+			articleUrl,
+			imageUrl
```
Comment (retrofox) [inline client/components/share/twitter-share-preview/index.jsx]: pfff trailing comma. feel free.

**Comment by retrofox** [comment]
Hi @jhnstn, how can I test this implementation?

**Comment by jhnstn** [comment]
@retrofox  just updated the description with test steps.

**Comment by jhnstn** [comment]
Changes are now in #12598

**Comment by retrofox** [comment]
> Changes are now in #12598
awesome

**Pull request closed without merging**